### ISSUE TITLE
Githug doesnt work

### ISSUE THREAD

**Issue by agderes** [Gazler/githug#337]
I have installed githug on my computer following the instructions. After installation, when I tried running the 'githug' command, I encountered an error.
![Screenshot 2023-06-21 095439](https://github.com/Gazler/githug/assets/56736620/5ab9ba43-4c71-4b61-a68f-4b9f2ba62cbb)

**Comment by VarunSingh2** [comment]
It seems that `exists` has been removed from ruby version 3.2.0 onwards. We may have to install ruby version below 3.2.0 for the time being.  Adding the [stack overflow link](https://stackoverflow.com/a/75353113 link)

**Comment by roxxrroy** [comment]
Then what is the solution for the above error ??

Should i install ruby version below 3.2.0 in my system @VarunSingh2 ,
Hey @agderes Did you found any solution for the above issue ??

**Comment by agderes** [comment]
> Then what is the solution for the above error ??
> 
> Should i install ruby version below 3.2.0 in my system @VarunSingh2 , Hey @agderes Did you found any solution for the above issue ??

Yes, i did install ruby version below 3.2.0 but dont remember which one.
